pkg/kubelet/kubelet_test.go

```diff
@@ -2609,6 +2610,41 @@ func TestSyncLabels(t *testing.T) {
 	}
 }
 
+func TestRetryRemovePodAfterSourcesReady(t *testing.T) {
```

Comment:
Does this test fail without your fix?

Reply:
Yea
=== RUN   TestRetryRemovePodAfterSourcesReady
    kubelet_test.go:2644: expected [12345678] kills, got []
--- FAIL: TestRetryRemovePodAfterSourcesReady (2.01s)